docs/src/components/CodeBlock.tsx

```diff
@@ -131,21 +145,20 @@ export const CodeBlock: React.FC<CodeBlockProps> = ({
           {({ className, style, tokens, getLineProps, getTokenProps }) => (
             <>
               <LiveProvider scope={{ ...Components, ...Icons }}>
-                <pre
-                  className={className + codeBoxStyles}
-                  style={{
-                    ...style,
-                  }}
+                <Box
+                  as="pre"
+                  css={{ ...style, ...codeBoxStyles }}
```

Comment:
in line 92 you add the styles the other way around, is this on purpose here?

Reply:
Oh no it wasn't on purpose. I will correct it.